src/ot_orchestration/dags/finngen_finemapping_ingestion.py

```diff
@@ -0,0 +1,71 @@
+"""Airflow DAG for the Preprocess part of the pipeline."""
+
+from __future__ import annotations
+
+from pathlib import Path
+from ot_orchestration.utils import common
+from airflow.models.dag import DAG
+from airflow.utils.trigger_rule import TriggerRule
+from ot_orchestration.utils.dataproc import (
+    delete_cluster,
+    submit_step,
+    create_cluster,
+    install_dependencies,
+)
+from airflow.models.baseoperator import chain
```

Comment:
```suggestion
from pathlib import Path

from airflow.models.baseoperator import chain
from airflow.models.dag import DAG
from airflow.utils.trigger_rule import TriggerRule

from ot_orchestration.utils import common
from ot_orchestration.utils.dataproc import (
    create_cluster,
    delete_cluster,
    install_dependencies,
    submit_step,
)
```

Sort imports similarly to the rest of the files.

Reply:
i did not realise, that we were missing the isort in ruff. Added it. Thanks for the point!